toolkit/tools/specreader/specreader.go

```diff
@@ -44,6 +45,7 @@ type parseResult struct {
 var (
 	app                     = kingpin.New("specreader", "A tool to parse spec dependencies into JSON")
 	specsDir                = exe.InputDirFlag(app, "Directory to scan for SPECS")
+	parseListFile           = app.Flag("parse-list", "Path to a list of SPECs to parse. If empty will parse all SPECs.").ExistingFile()
```

Comment:
Do we want to rename this to specListFile? Accordingly, we'll rename parse-list to spec-list

Reply:
Sure, I can rename that if it reads better. This is in contrast to the existing `packList` variables used in the srpm packer (set via SRPM_PACK_LIST=...)